mobile/voice_journal_app/lib/home.dart

```diff
@@ -1,77 +1,115 @@
-                  color: Colors.black, 
-                ),
-              ),
-              SizedBox(height: 20), // Adds space between the title and the first container
-              Container(
-                height: 200,
-                decoration: BoxDecoration(
-                  color: AppColors.secondaryColor, // Background color
-                  borderRadius: BorderRadius.circular(10), // Rounds corners
+          padding: const EdgeInsets.symmetric(horizontal: 16.0), // Horizontal padding for white space
+          child: Container(
+            padding: const EdgeInsets.only(bottom: 60.0, top: 90.0), // Adjusted to accommodate FAB and title
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.end, // Aligns children to the bottom
+              crossAxisAlignment: CrossAxisAlignment.stretch, // Stretches children horizontally to match the column width
+              children: <Widget>[
+                const Text(
+                  'Emoz',
+                  textAlign: TextAlign.center, //
+                  style: TextStyle(
+                    fontSize: 24, 
+                    fontWeight: FontWeight.bold, 
+                    color: Colors.black, 
+                  ),
                 ),
-                child: Center(child: Text('Emotion Stats')), // Placeholder text
-                margin: EdgeInsets.only(bottom: 20, top: 20), // Space between the rectangles and additional top padding
-              ),
-              Expanded( // 
-                child: Container(
+                const SizedBox(height: 20), // Adds space between the title and the first container
+                Container(
+                  height: 200,
                   decoration: BoxDecoration(
-                    color: AppColors.primaryColor, // Background color of the rectangle
-                    borderRadius: BorderRadius.circular(10), // Rounded corners
-                  ),
-                  child: ListView.builder( 
-                    itemCount: 10, // Number of items in the list, for demonstration
-                    itemBuilder: (context, index) => ListTile(
-                      title: Text('Item $index'), // Example list item
+                    color: AppColors.secondaryColor, // Background color
+                    borderRadius: BorderRadius.circular(10), // Rounds corners
+                  ), // Placeholder text
+                  margin: const EdgeInsets.only(bottom: 20, top: 20),
+                  child: const Center(child: Text('Emotion Stats')), // Space between the rectangles and additional top padding
+                ),
+                Expanded( // 
+                  child: Container(
+                    decoration: BoxDecoration(
+                      color: AppColors.primaryColor, // Background color of the rectangle
+                      borderRadius: BorderRadius.circular(10), // Rounded corners
+                    ),
+                    child: ListView.builder( 
```

Comment:
Let's set up a function that returns a `Future` and in that function we can open the box and return a list of recordings. Then here we can have the child be a `FutureBuilder` which will have the `ListView` builder as a child. This will allow us to load the recordings in the background, not having to await anything, and to open the box in that function instead of in the `Main` class. Take a look at FutureBuilders: https://www.educative.io/answers/what-is-a-futurebuilder

Reply:
Take a look at how Anika did it for the Stats page: https://github.com/LaurierCS/POD1/blob/895ca5886af919ce191a7d5d0decf0608cd65aac/mobile/voice_journal_app/lib/stats.dart